Khan/wonder-stuff
# [fei4220.1.changesets] Setup changeset publishing and force Node 12 or above

## Summary:
This sets up changesets on Wonder Stuff and also sets a minimum engine of Node 12 on the root package.json.

Addresses FEI-4220 and FEI-4349

Issue: FEI-4220

## Test plan:
1. Try running `yarn` in a Node version less than 12 and see the given error
2. Try running `yarn` in a Node version greater than or equal to 12 and see things work
3. Putting this PR up should see the changeset bot comment on our changeset
4. Merging this PR should cause a Version Packages PR to get made
5. Merging the Version Packages PR should cause a new NPM release of all packages as a major version and post update notices to #frontend-infra-mobile and #frontend-infra-web
